app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -288,6 +290,15 @@ object TelemetryWrapper {
         averageTime = 0.0
     }
 
+    private var shouldSendMobileMetrics = false
+
+    @JvmStatic
+    fun addMobileMetricsPing(mobileMetrics: JSONObject) {
+        TelemetryHolder.get().addPingBuilder(TelemetryMobileMetricsPingBuilder(mobileMetrics,
+                TelemetryHolder.get().configuration))
+        shouldSendMobileMetrics = true
+    }
```

Comment:
Can't we just always add the ping builder above? We just do not need to queue a ping always.

Reply:
The TelemetryMobileMetricsPingBuilder needs the JSONObject passed in when it's created + added and I'm not sure where is the right place to get the reference to GeckoView that I need to get the JSONObject. That's why I originally did it in WebFragment so I would have the IWebView reference there.